src/swap/changelly.js

```diff
@@ -153,6 +160,7 @@ export function makeChangellyPlugin (
       userSettings: Object | void
     ): Promise<EdgeSwapPluginQuote> {
       if (
+        // if either currencyCode is invalid *and* doesn't have a transcription
```

Comment:
This seems like it will always throw - I don't see where we use the `CURRENCY_CODE_TRANSCRIPTION` in this check. Actually, since `CURRENCY_CODE_TRANSCRIPTION` handles this case, I think the entire check can just go away.